grin/src/Transformations/ExtendedSyntax/Names.hs

```diff
@@ -128,6 +128,7 @@ mapNameDefExpM f = \case
   EBind leftExp (VarPat var) rightExp -> do EBind leftExp <$> (VarPat <$> f var) <*> pure rightExp
   EBind leftExp (AsPat var val) rightExp -> EBind leftExp <$> (AsPat <$> f var <*> mapNamesValM f val) <*> pure rightExp
   Alt cpat body               -> Alt <$> mapNamesCPatM f cpat <*> pure body
```

Comment:
Is there any use-case left for the Alt? Should we remove it soon?

Reply:
Yes, we should remove it in the next PR.

Reply:
I already removed it on another branch: [32-alt-remove-unnamed](https://github.com/grin-compiler/grin/tree/32-alt-remove-unnamed)

Reply:
I already removed it on another branch: [32-alt-remove-unnamed](https://github.com/grin-compiler/grin/tree/32-alt-remove-unnamed)